src/components/Database/DataViewer/ImportDialog.module.scss

```
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.container {
+  position: relative;
+
+  :global .Callout {
```

Does this need to be global? If it does, maybe it could live higher-up in the overrides?

---

`:global` means that it will not prefix that immediate class, so it's still local to this scoped component... is weird. if you nest styles in scss + modules it prefixes them all!

Without global: `.container_xy .container_xy_Callout`, which seems totally useless!

With global: `.container_xy .Callout`...

---

https://github.com/webpack-contrib/css-loader/issues/346

I preferred this over adding a specific class to `<Callout>` just to add margin... 

It's really the container who cares about the margin.

I don't love this..

In fact this is making me lean towards [emotion styled-components](https://emotion.sh/docs/nested), which support style encapsulation AND nice nesting ergonomics:


```ts
const Container = styled.div`
  color: red;

  // nesting other raw styles works fine
  .Callout {
    margin-bottom: 24px;
  }
  
  // nesting other styled components:
  ${SomeOtherComponent} {
    margin: 8px;
  }
`;
```

---

Tim is right about this `:global` usage. However, this line can be made much less ambiguous:
```suggestion
  :global(.Callout) {
```

This clarifies that the class name `Callout` is global while the whole rule is not.